webextensions/api/browserAction.json

```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction",
           "support": {
             "chrome": {
-              "version_added": "≤58",
+              "version_added": "4",
```

Comment:
✅ Added in https://github.com/chromium/chromium/commit/ec9ac0df1c02dc01550b1020ef7e74bd795a2008, which landed in [4.0.221.8](https://github.com/chromium/chromium/releases/tag/4.0.221.8) before Chrome 4.0.223.